ts/components/wallet/CanelPaymentLink.tsx

```diff
@@ -0,0 +1,50 @@
+/**
+ * description
+ */
+
+import { Text } from "native-base";
+import * as React from "react";
+import { NavigationScreenProp, NavigationState } from "react-navigation";
+import { connect } from "react-redux";
+import I18n from "../../i18n";
+import { Dispatch } from "../../store/actions/types";
+import { paymentRequestCancel } from "../../store/actions/wallet/payment";
+
+type OwnProps = Readonly<{
+  navigation: NavigationScreenProp<NavigationState>;
+}>;
+
+type ReduxMappedDispatchProps = Readonly<{
+  cancelPayment: () => void;
+}>;
+
+type Props = OwnProps & ReduxMappedDispatchProps;
+
+class GoBackLink extends React.PureComponent<Props> {
+  public render(): React.ReactNode {
+    return (
+      <Text
```

Comment:
is this supposed to be a button?

Reply:
Not being a button in the invisio, I implemented a bolded link.
Here the invisio:
<img width="362" alt="image" src="https://user-images.githubusercontent.com/38431762/44976338-3c1ea880-af65-11e8-8e20-b95a229ef0cb.png">


Here the real implementation:
<img width="356" alt="image" src="https://user-images.githubusercontent.com/38431762/44976147-8eab9500-af64-11e8-9d71-c52209082a4d.png">


Do you prefer a button instead?